packages/account-v2/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { PropsWithChildren, useState } from 'react';
```

Comment:
is this accordian different from https://deriv-ui.pages.dev/?path=/docs/components-accordion--docs

Reply:
Will replace this. Thanks